### ISSUE TITLE
textDecoration only supports textDecorationLine

### ISSUE THREAD

**Issue by davesnx** [giraud/bs-css#258]
Hi @giraud 

Currently `textDecoration` (https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration) supports only `textDecorationLine` (https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration)

I'm not sure how to map a shorthand property that has been used as a single property, and the cleanest solution is to break the textDecoration and bump a version. WDYT?

**Comment by giraud** [comment]
On vacation until end of august

**Comment by giraud** [comment]
right, this is totally incorrect. I don't see how we can improve the situation without a breaking change.
